### DISCUSSION TITLE
Scored filtering

### DISCUSSION THREAD

**Discussion by Kerollmops** [meilisearch/product#649] [Feedback & Feature Proposal]
Filter scoring can rank records according to how well or poorly they match a set of filters. The filter syntax is part of #648. When a document matches multiple conditions it can either get the sum of the different conditions or the highest score (default).

Here is an example syntax [from Algolia's feature](https://www.algolia.com/doc/guides/managing-results/refine-results/filtering/in-depth/filter-scoring/):

```sql
(company:Google<score=3> OR company:Amazon<score=2> OR company:Facebook<score=1>)
```

**Comment by CaroFG** [comment]
Hey there!
We just published a workaround guide on our blog: [how to boost your search results with filter scoring](https://blog.meilisearch.com/filter-scoring-guide/).
We hope it's useful!
